overlay.yaml

```diff
@@ -0,0 +1,1057 @@
+overlay: 1.0.0
```

Comment:
Does this file need to be at the root of the repo? Why not inside `packages/sdk`?

Reply:
Good catch - updated